### PR TITLE
fix(scss) timeline and pagination imports

### PR DESCRIPTION
**Pull request by lucienbertin**
both componetns werent loaded when importing the overridable version of lf/scss

### CODE REVIEW

**Review by sanlucca — APPROVED** [review]

**Review by RipripKipkip — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1178

File: packages/scss/src/components/_timeline.scss [hunk 1]
```diff
@@ -98,9 +98,9 @@
 
 		.gauge, .progress {
 			position: absolute;
-			left: (_theme("spacings.smallest", true) / -2);
+			left: cals(#{_theme("spacings.smallest")} / -2);
```
Comment (lucienbertin) [inline packages/scss/src/components/_timeline.scss:101]: @sanlucca do you prefer a `.5 *` instead of a `/ 2` ?

Comment (sanlucca) [reply packages/scss/src/components/_timeline.scss:101]: no I prefer / 2 * -1 and calc instead of cals

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1178

**Review by sanlucca — APPROVED** [review]

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1178

**Pull request merged**